src/components/DocumentationTopic.vue

```diff
@@ -544,7 +544,8 @@ export default {
       return declarations.length ? declarations[0].declarations.some(decl => Object.prototype.hasOwnProperty.call(decl, 'otherDeclarations')) : false;
     },
     declListToggleText({ declListExpanded }) {
-      return declListExpanded ? 'Hide other declarations' : 'Show all declarations';
+      return declListExpanded ? `${this.$t('verbs.hide')} ${this.$t('declarations.other-declarations')}`
```

Comment:
I'm not an expert, but we may want to use a single string for this instead of splitting it up into 2 strings for the noun and verbs and concatenating them. The reason is that we can't guarantee that the verb would always come before the noun in all languages. (@marinaaisa please correct me if this is wrong)

Reply:
Yep, it should be a single sentence.